pikaraoke/app.py

```diff
@@ -550,7 +550,10 @@ def info():
     url = k.url
 
     # cpu
-    cpu = str(psutil.cpu_percent()) + "%"
+    try:
+        cpu = str(psutil.cpu_percent()) + "%"
+    except:
+        cpu = "Android CPU usage query unsupported."
```

Comment:
This might not be specific to android, `CPU usage query unsupported` will be more universal